packages/edit-navigation/src/components/inserter-sidebar/index.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+import {
+	__experimentalLibrary as Library,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import {
+	useViewportMatch,
+	__experimentalUseDialog as useDialog,
+} from '@wordpress/compose';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as editNavigationStore } from '../../store';
+import { useNavigationEditorRootBlock } from '../../hooks';
+
+function InserterSidebar() {
+	const SHOW_PREVIEWS = false;
```

Comment:
Do we still need this constant? We can pass `false` directly into `showInserterHelpPanel`. But If you preferer having a constant, we should move it outside of the function. No need to recreate it on every rerender.

Reply:
I would like the constant purely as a way to make the code more self documenting. I'll move outside.

Reply:
✅